### PR TITLE
Introducing Imagenette 160 as a test dataset

### PR DESCRIPTION
**Pull request by QuentinDuval**
Imagenette allows for quick iterations on algorithms and has been created for that purpose: https://github.com/fastai/imagenette.

Imagenette is directly pluggable as a "disk_folder" and no code change is needed. This PR therefore only contains two examples configurations for Imagenette 160 (one for pre-training with SimCLR, the other for linear evaluation).

We found that Imagenette is great to test idea, and has been used as well for publishing implementations and helping reproduce them on small datasets first (for instance, Deepmind used it to help reproducing BYOL: https://github.com/deepmind/deepmind-research/tree/master/byol)

### CODE REVIEW

**Comment by prigoyal** [comment]
this looks awesome! let's land it soon :) can you import the PR to phabricator ? :)

**Review by prigoyal — APPROVED** [review]

File: configs/config/benchmark/linear_image_classification/imagenette/eval_resnet_8gpu_transfer_imagenette_160.yaml [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# @package _global_
```
Comment (prigoyal) [inline configs/config/benchmark/linear_image_classification/imagenette/eval_resnet_8gpu_transfer_imagenette_160.yaml:1]: is it possible to move this file to `benchmark/linear_image_classification/datasets/imagenette_160.yaml` ? and only keep the relevant changes in the file ? we can compose the main benchmark config and dataset config together :)

File: configs/config/pretrain/simclr/simclr_1node_resnet_imagenette_160.yaml [hunk 2]
```diff
@@ -0,0 +1,111 @@
+# @package _global_
```
Comment (prigoyal) [inline configs/config/pretrain/simclr/simclr_1node_resnet_imagenette_160.yaml:1]: is it possible to move this to `configs/config/pretrain/simclr/datasets/imagenette_160.yaml` and keep only the relevant changes in the yaml file?

Comment (QuentinDuval) [reply configs/config/pretrain/simclr/simclr_1node_resnet_imagenette_160.yaml:1]: As discussed together, the idea of Imagenette is to help quickly iterate and debug issues. This is not really for pretraining not benchmarking in the real world, so I will move them to "debugging/pretrain/" and "debugging/benchmark/" as suggested.

**Pull request closed without merging**